test-support/trie-bench/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.33.0] - 2023-02-20
```

Comment:
```suggestion
## [0.34.0] - 2023-02-20
```

Also, why Feb 20?

Reply:
Thanks (I can't read the date on my pc properly :facepalm:  :)

Reply:
The changelog still says `0.33.0`. 😛